test/blackbox-tests/test-cases/dune-ppx-driver-system/driver-tests/dune

```diff
@@ -1,10 +1,3 @@
-; No driver found
-(library
- (name foo1)
- (public_name foo.1)
- (modules foo1)
- (preprocess (pps)))
-
```

Comment:
Why is this test changed? (Sorry if I missed something).

Reply:
This is because the check for en empty `(pps)` now takes place during parsing time, so if we leave that in there it keeps the rest of the tests from running. Instead I moved this test to a different stand-alone test at the top of the `run.t` file (in the parent directory of this file).